### ISSUE TITLE
Any plans to fork FaceFusion instead since roop is dead?

### ISSUE THREAD

**Issue by UberGuidoZ** [C0untFloyd/roop-unleashed#180]
Since the main dev of roop has [decided to close](https://github.com/s0md3v/roop#this-project-has-been-discontinued) to the project and no longer develop it, are there plans to change the main fork over to the successor, [FaceFusion](https://github.com/facefusion/facefusion)?

**Comment by C0untFloyd** [comment]
No. The last month this fork had so many advancements over the original roop and face fusion, I don't even think of it as a fork anymore. If there is something face fusion is doing better or a bug fix which affects both, I could still do some cherry-picking. I would rather fork from Rope RIP as I feel it had the better features and more advanced stuff.
